common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Agent/Default/Worker.cs

```diff
@@ -68,6 +68,14 @@ public class Worker : IWorker, IDisposable {
             _agentConfigProvider.OnConfigUpdated += (s, e) => {
```

Comment:
We need to cleanup the callback that we added on `Dispose()`.

Reply:
added event handler in StartAsync() and removed it in StopAsync()